aesara/link/jax/jax_linker.py

```diff
@@ -59,6 +59,7 @@ def create_jax_thunks(self, compute_map, storage_map):
             n for n, i in enumerate(self.fgraph.inputs) if isinstance(i, Constant)
         ]
 
+        storage_map = jaxify(storage_map)
```

Comment:
I'm not sure if this is a sound point of entry.  Under this approach we're not dispatching on the actual objects that we're ultimately converting: i.e. Aesara `Type`s.  It also adds machinery for traversing storage maps, which is somewhat tangential to the overall conversion process.

Reply:
How about the current entry point ?